scripts/microbe_stage/microbe.as

```diff
@@ -433,14 +434,17 @@ class MicrobeSystem : ScriptSystem{
             microbeComponent.movementFactor =  microbeComponent.movementFactor/ENGULFING_MOVEMENT_DIVISION;
         }
 
+        microbeComponent.escapeInterval = microbeComponent.escapeInterval + logicTime;
+
         if(microbeComponent.isBeingEngulfed){
             microbeComponent.movementFactor =  microbeComponent.movementFactor/ENGULFED_MOVEMENT_DIVISION;
             //LOG_INFO("doing engulf damage");
             MicrobeOperations::damage(world,microbeEntity,ENGULF_DAMAGE/logicTime,
                 "isBeingEngulfed - Microbe.update()s");
             microbeComponent.wasBeingEngulfed = true;
+            microbeComponent.escapeInterval = 0;
             // Else If we were but are no longer, being engulfed
-        } else if(microbeComponent.wasBeingEngulfed && !microbeComponent.isBeingEngulfed){
+        } else if(microbeComponent.wasBeingEngulfed && !microbeComponent.isBeingEngulfed && microbeComponent.escapeInterval > CREATURE_ESCAPE_INTERVAL){
```

Comment:
I feel like this is the wrong place to prevent the engulfing from stopping. This interval check being here prevents `removeEngulfedEffect` from being called.